runtime/src/identity.rs

```diff
@@ -686,6 +739,15 @@ impl<T: Trait> Module<T> {
             }
         }
     }
+
+    /// It set/reset the current identity.
+    pub fn set_current_did(did_opt: Option<IdentityId>) {
```

Comment:
This shouldn't be public. No other module (or smart extension) should be able to call this.

Reply:
It is used by `UpdateDid`. BTW we can use `pub (crate)` visibility if we create the `identity` rust module and move `UpdateDid` to this new module.
I am going to add a comment in [MESH-391](https://polymath.atlassian.net/browse/MESH-391)